client/packages/system/src/MasterList/DetailView/ContentArea.tsx

```diff
@@ -1,16 +1,64 @@
 import React from 'react';
-import { DataTable, useTranslation } from '@openmsupply-client/common';
+import { DataTable, useColumns, useTranslation, useUrlQueryParams } from '@openmsupply-client/common';
 import { useMasterList } from '../api';
+import { MasterListLineFragment } from '../api/operations.generated';
 
 export const ContentArea = () => {
   const t = useTranslation();
-  const { lines, columns } = useMasterList.line.rows();
+  const { data, isLoading, isError } = useMasterList.line.rows();
+  const {
+    updateSortQuery,
+    updatePaginationQuery,
+    queryParams: { sortBy, page, first, offset },
+  } = useUrlQueryParams({
+    initialSort: { key: 'name', dir: 'asc' },
```

Comment:
the name column is sorting, but not shown as the sort column!
```suggestion
    initialSort: { key: 'itemName', dir: 'asc' },
```